src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/CitizenClaimCreatedEvent.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.cmc.claimstore.events.claim;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Value;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import uk.gov.hmcts.cmc.claimstore.events.ClaimCreationEvent;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Getter
+@EqualsAndHashCode(callSuper = false)
```

Comment:
`callSuper = false` is not needed as this is the default

Reply:
Done